lib/matplotlib/backends/backend_ps.py

```diff
@@ -39,17 +39,9 @@
 debugPS = False
 
 
-@_api.deprecated("3.7")
-class PsBackendHelper:
-    def __init__(self):
-        self._cached = {}
-
-
 @_api.caching_module_getattr
 class __getattr__:
     # module-level deprecations
-    ps_backend_helper = _api.deprecated("3.7", obj_type="")(
-        property(lambda self: PsBackendHelper()))
     psDefs = _api.deprecated("3.8", obj_type="")(property(lambda self: _psDefs))
```

Comment:
I thinks this line here should be removes also as it uses _api.depreciated

Reply:
Only the deprecations for 3.7 are expiring now, we leave two minor releases for deprecations, so the 3.8 ones will go away for 3.10.